Jenkinsfile

```diff
@@ -326,14 +342,15 @@ pipeline {
         axes {
           axis {
             name 'CASSANDRA_VERSION'
-            values '3.11',    // Current Apache Cassandra
-                   '4.0',     // Development Apache Cassandra
-                   'dse-5.1.35', // Legacy DataStax Enterprise
-                   'dse-6.8.30' // Development DataStax Enterprise
+            values '3.11',    // Previous Apache Cassandra
+                   '4.1.0',    // Previous Apache Cassandra 
+                   '5.0-beta1', // Current Apache Cassandra
```

Comment:
I changed `4.1.0` to `4.1`, however, I am afraid we still need `5.0-beta1` because `ccm create cl -v 5.0` only works with java 17 for now.

Reply:
This is presumably because 5.0 has to be... built from source?  If that's the case let's wait until 5.0.1 is out since that should be happening very, very soon.

Reply:
Cassandra 5.0.1 is [now released](https://lists.apache.org/thread/sq1v6fwpk53yb33h14tbfxq6fo9kvsjt).  It looks like the content is propagating through the CDN at the moment but it should be showing up pretty shortly.